instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Comment from './Comment'
+
+const CommentSection = props => {
+    console.log(props.comments);
+    return <div>{props.comments.map((c, i) =>
+         <Comment username={c.username} key={i} text={c.text} />
```

Comment:
I touched on this briefly this week, but `index` is not always the best option for keys when rendering a list of elements in React. the reason being is that if you later need to modify or resort the data in any way, it will break your code: you can either manually add an `id` property to each comment/post before originally setting it on state to use here instead, use an id-generating library like `uuid` to help do that with less work, or pick a unique property already on each comment object. this won't matter here until you attempt to add a `delete comment` functionality as a stretch, but it's something to keep in mind